packages/astroport-governance/src/voting_escrow.rs

```diff
@@ -251,3 +254,11 @@ pub fn get_lock_info(
     )?;
     Ok(lock_info)
 }
+
+/// ## Description
+/// Query blacklisted voters from the voting escrow contract.
+pub fn get_blacklisted_voters(querier: QuerierWrapper, escrow_addr: &Addr) -> StdResult<Vec<Addr>> {
```

Comment:
I think we need to add pagination here as we can hit the maximum terra msg limit if there are too many blacklisted addresses.